lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,9 +289,9 @@ def main():
     new_path = module.params.get('new_path')
     dup_ok = module.params.get('dup_ok')
     if state == 'present' and not new_name and not new_path:
-        cert = module.params.get('cert')
-        key = module.params.get('key')
-        cert_chain = module.params.get('cert_chain')
+        cert, key, cert_chain = load_data(cert=module.params.get('cert'),
```

Comment:
I stand corrected, uploading certs/keys in the AWS GUI without the PEM headers does not work.